### PR TITLE
[Checkout page] Price is lower if certificate was used in previous order

### PR DESCRIPTION
**Pull request by MickhaelSh**
Changed deleting products method, changed saving and showing phone numbers

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by tarasSanotskyi — DISMISSED** [review]

File: src/containers/checkout/checkout-form/checkout-form.js [hunk 1]
```diff
@@ -145,12 +137,7 @@ const CheckoutForm = ({ cartItems, promoCode, certificate, handleCashPayment })
 
   useEffect(() => {
     if (userData) {
-      setInitialValues(
-        updateInitialValues(
-          { ...userData, phoneNumber: userData.phoneNumber.slice(3) },
-          deliveryType
-        )
-      );
+      setInitialValues(updateInitialValues({ ...userData }, deliveryType));
```
Comment (LubomyrKashuba) [inline src/containers/checkout/checkout-form/checkout-form.js:140]: can you use just "updateInitialValues(userData, deliveryType)"?

Comment (MickhaelSh) [reply src/containers/checkout/checkout-form/checkout-form.js:140]: no, it's just a helper function that returns an object, and we need to save it

**Review by IrynaMel — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by IrynaMel — APPROVED** [review]

File: src/pages/profile-page/profile-page.js [hunk 2]
```diff
@@ -49,14 +49,12 @@ const ProfilePage = () => {
   }));
 
   const handleSaveUser = ({ firstName, lastName, email, phoneNumber, ...address }) => {
-    if (phoneNumber === null) {
-      phoneNumber = '';
-    }
+    const userPhoneNumber = phoneNumber || '';
```
Comment (olga-gchak) [inline src/pages/profile-page/profile-page.js:52]: Why do you add this changes?

I think we already have it in scope of this PR:
https://github.com/ita-social-projects/horondi_client_fe/pull/2294/files

**Review by olga-gchak — CHANGES_REQUESTED** [review]

File: src/pages/profile-page/profile-page.js (hunk 2 above)
Comment (MickhaelSh) [reply src/pages/profile-page/profile-page.js]: Miscommunication, fixed

**Review by byavdoshniak — APPROVED** [review]

**Review by Pipuss — APPROVED** [review]

**Review by Yevhendzyuba — APPROVED** [review]

**Review by olga-gchak — APPROVED** [review]

**Pull request merged**